lib/mysqlDaoQueryHelper.js

```diff
@@ -58,6 +58,23 @@ class MysqlDaoQueryHelper {
     }
 
     /**
+     * @param {object} upsertArgs
+     * @returns {string}
+     */
+    upsert(upsertArgs) {
```

Comment:
another suggestion: i can think of a few use cases where it would be useful to know if the row was actually inserted or updated, right now the returned id wouldn't indicate either